manifests/init.pp

```diff
@@ -115,12 +115,23 @@
     refreshonly => true,
   }
 
+  file{[
+    '/usr/local',
+    '/usr/local/sbin',
+  ]:
+    ensure => directory,
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0755',
+  }
+
   # Used in letsencrypt::certonly Exec["letsencrypt certonly ${title}"]
   file { '/usr/local/sbin/letsencrypt-domain-validation':
     ensure => file,
     owner  => 'root',
     group  => 'root',
     mode   => '0500',
     source => "puppet:///modules/${module_name}/domain-validation.sh",
+    require=> File['/usr/local/sbin'],
```

Comment:
This hasn't been needed since Puppet 2.7 and file autorequires parent directoryies.